File: CodeWars/Seven_Segment_Display/sol.cpp
#include <bits/stdc++.h>
using namespace std;

int32_t main(){
   int n;
   cin >> n;

   cout << "THE MAXIMUM NUMBER FORMED IS ";
   if (n & 1) {
      cout << 7;
      for (int i = 0; i < n - 3; i += 2) {
         cout << 1;
      }
      cout << '\n';
   } else {
      for (int i = 0; i < n; i += 2) {
         cout << 1;
      }
      cout << '\n';
   }

   cout << "THE MINIMUM NUMBER FORMED IS ";
   string res = "";
   while (n >= 6) {
      res += "0";
      n -= 6;
   }

   if (n == 1) {
      res.pop_back();
      res += '8';
   } else if (n == 2) {
      res += '1';
   } else if (n == 3) {
      res += '7';
   } else if (n == 4) {
      res += '4';
   } else if (n == 5) {
      res += '2';
   }

   cout << res << '\n';
   return 0;
}

File: CodeWars/Seven_Segment_Display/sol.py
n = int(input())

if n % 2 == 0:
   print ("THE MAXIMUM NUMBER FORMED IS " + '1' * (n // 2))
else :
   print ("THE MAXIMUM NUMBER FORMED IS " + '7' + '1' * ((n - 3) // 2))

Min = ""
while n >= 6:
   Min += "0"
   n -= 6

if n == 1:
   Min = Min[:-1]
   Min += '8'
elif n == 2:
   Min += '1'
elif n == 3:
   Min += '7'
elif n == 4:
   Min += '4'
elif n == 5:
   Min += '2'

print ("THE MINIMUM NUMBER FORMED IS " + Min)